mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_service_test.go

```diff
@@ -0,0 +1,100 @@
+package apphub_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestDataSourceApphubDiscoveredService_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceApphubDiscoveredService_basic(context),
+			},
+		},
+	})
+}
+
+func testDataSourceApphubDiscoveredService_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(
+		`
+resource "google_project" "service_project" {
+	project_id ="<%= ctx[:vars]['service_project_attachment_id'] %>"
+	name = "Service Project"
+	org_id = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_apphub_service_project_attachment" "service_project_attachment" {
+  service_project_attachment_id = google_project.service_project.project_id
+}
+
+# discovered service block
+data "google_apphub_discovered_service" "catalog-service" {
+  provider = google
+  location = "us-east1"
+  # ServiceReference | Application Hub | Google Cloud
+  # Using this reference means that this resource will not be provisioned until the forwarding rule is fully created
+  service_uri = "//compute.googleapis.com/${google_compute_forwarding_rule.forwarding_rule.id}"
+	depends_on = [google_apphub_service_project_attachment.service_project_attachment]
+}
+
+# VPC network
+resource "google_compute_network" "ilb_network" {
```

Comment:
Have we enabled compute API for service_project?

Reply:
done